lib/webapp.js

```diff
@@ -671,6 +679,16 @@ const staticHandlers = {
     });
     return router;
   },
+  
+  raft(webApp) {
+    const router = express.Router();
+    webApp.expressWs.applyTo(router);
+    router.ws('/raft', (ws, req) => {
+      console.log(`client connected to raft service`);
```

Comment:
will need to convert this to use of the logger

Reply:
This message must be removed.